docs/bib/use_citations.bib

```diff
@@ -75,12 +75,16 @@ @article{Alguero:2020grj
 }
 
 % 2020-08-14
-@Booklet{ATLAS-CONF-2020-046,
-    author         = "{ATLAS Collaboration}",
-    title          = "{Search for new phenomena in events with two opposite-charge leptons, jets and missing transverse momentum in \(pp\) collisions at \(\sqrt{s} = 13\,\text{TeV}\) with the ATLAS detector}",
-    howpublished   = "{ATLAS-CONF-2020-046}",
-    url            = "https://cds.cern.ch/record/2728056",
-    year           = "2020",
+@article{Aad:2021hjy,
+    author = "Aad, Georges and others",
+    collaboration = "ATLAS",
```

Comment:
```suggestion
    author         = "{ATLAS Collaboration}",
```

Can we keep the author as ATLAS? I never really find it helpful when we have the author list and collaobration split out like this.